easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.5-foss-2016a-mpi.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
+name = 'IQ-TREE'
+version = '1.5.5'
+versionsuffix = "-mpi"
+
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+github_account = 'Cibiv'
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+
+configopts = '-DIQTREE_FLAGS="mpi omp" -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx'
```

Comment:
don't hardcode these compiler (wrapper) commands, enable the `usempi` toolchain option instead:

```python
toolchain = {'name': 'foss', 'version': '2016a'}
toolchainopts = {'usempi': True}